pandapipes/converter/stanet/valve_pipe_component/create_valve_pipe.py

```diff
@@ -0,0 +1,94 @@
+from pandapipes.component_models.component_toolbox import add_new_component
+from pandapipes.pandapipes_net import pandapipesNet
+from pandapower.auxiliary import get_free_id, _preserve_dtypes
+from pandapipes.converter.stanet.valve_pipe_component.valve_pipe_component import ValvePipe
+
+
+def create_valve_pipe(net, from_junction, to_junction, length_km, diameter_m, opened=True,
+                      loss_coefficient=0, sections=1, std_type=None, k_mm=0.15e-3,
+                      max_vdot_m3_per_s=100, max_v_m_per_s=None, alpha_w_per_m2k=0., qext_w=0.,
+                      name=None, index=None, in_service=True, type="valve_pipe", **kwargs):
+    """
+    Creates a valve pipe element in net["valve_pipe"] from valve pipe parameters.
+
+    :param net: The net within this pipe should be created
+    :type net: pandapipesNet
+    :param from_junction: ID of the junction on one side which the pipe will be connected with
+    :type from_junction: int
+    :param to_junction: ID of the junction on the other side which the pipe will be connected with
+    :type to_junction: int
+    :param length_km: The pipe length in km
+    :type length_km: float
+    :param diameter_m: The pipe diameter in m, for a start only circular pipes are supported
+    :type diameter_m: float
+    :param opened: Flag to show if the valve is opened and allows for fluid flow or if it is closed\
+            to block the fluid flow.
+    :type opened: bool, default True
+    :param loss_coefficient: The pressure loss coefficient introduced by the valve shape
+    :type loss_coefficient: float, default 0
+    :param sections: The number of internal pipe sections. Important for gas and temperature\
+            calculations, where variables are dependent on pipe length.
+    :type sections: int, default 1
+    :param std_type:
+    :type std_type: str, default None
+    :param k_mm:
+    :type k_mm: float, default 0.15e-3
+    :param max_vdot_m3_per_s:
+    :type max_vdot_m3_per_s: float, default 100
+    :param max_v_m_per_s:
+    :type max_v_m_per_s: ?, default None
```

Comment:
What is the purpose of these parameters? If they are simply there to transfer information from the Stanet net to the respective table and otherwise ignored by the pipeflow etc. this should be stated here

Reply:
@SimonRubenDrauz can you comment on that? I think that this is rather a remnant of the STANET model? Suppose it doesn't really add value to our model, does it?